linux/Jenkinsfile

```diff
@@ -4,4 +4,7 @@
 
 @Library('lsdev-pipeline-library') _
 
-keymanPackaging
+keymanPackaging {
+	distributionsToPackage = 'xenial bionic'
```

Comment:
Just checking. So this will fix the Jenkins error?
```
No such property: supportedDistros for class: keymanPackaging
```

Can you also add a note in the Linux readme so we'll remember to update this when fossa comes out?

Reply:
Yes, I think this will fix the error.

I'll update the Linux readme later after we get things working. There's no obvious place that I can see where that information should go, and it seems there are several sections that look like they might have to be updated.